awx/main/migrations/0208_workflow_approval_notifications.py

```
@@ -0,0 +1,20 @@
+from django.db import migrations, models
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0185_previous_migration_name_here'),  # Replace with actual latest migration file name
+    ]
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)